_rules/video-only-as-media-alternative-fd26cf.md

```diff
@@ -30,11 +30,9 @@ All the information contained in each target element is available as text (direc
 
 ## Expectation 2
 
-Each target element is labeled as a video alternative for text on the page.
+Each target element is labeled as a video alternative for text on the page. This label is [visible][] and [included in the accessibility tree][].
```

Comment:
```suggestion
Each target element is identified as a video alternative for text on the page by a [visible][] label that is [included in the accessibility tree][].
```

Reply:
Another option is "Each target element is labeled as a video alternative for text on the page by a [visible][] identifier that is [included in the accessibility tree][].

This second option doesn't use "label" so the note wouldn't be necessary, but not sure if "identifier" is a term that's been used in rules before.

Reply:
I agree this can be amended, which is the preferred result as I will need to update the other file to match.

Reply:
I think I prefer Kathy's version of this.